### PR TITLE
refactor: minor improvements for breadcrumbs

### PR DESCRIPTION
**Pull request by lex111**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

- Improved vertical alignment

image

- Fix a11y related error "Links must have discernible text"
- Make translatable messages for aria-label attributes

### CODE REVIEW

File: packages/docusaurus-theme-translations/locales/zh-Hans/theme-common.json [hunk 1]
```diff
@@ -33,6 +33,8 @@
   "theme.common.headingLinkTitle": "标题的直接链接",
   "theme.common.skipToMainContent": "跳到主要内容",
   "theme.docs.DocCard.categoryDescription": "{count} 个项目",
+  "theme.docs.breadcrumbs.home": "Home page",
+  "theme.docs.breadcrumbs.navAriaLabel": "Breadcrumbs",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-translations/locales/zh-Hans/theme-common.json:37]:
```suggestion
  "theme.docs.breadcrumbs.home": "主页面",
  "theme.docs.breadcrumbs.navAriaLabel": "页面路径",
```

File: packages/docusaurus-theme-translations/locales/zh-Hant/theme-common.json [hunk 2]
```diff
@@ -33,6 +33,8 @@
   "theme.common.headingLinkTitle": "標題的直接連結",
   "theme.common.skipToMainContent": "跳至主要内容",
   "theme.docs.DocCard.categoryDescription": "{count} 個項目",
+  "theme.docs.breadcrumbs.home": "Home page",
+  "theme.docs.breadcrumbs.navAriaLabel": "Breadcrumbs",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-translations/locales/zh-Hant/theme-common.json:37]:
```suggestion
  "theme.docs.breadcrumbs.home": "主頁面",
  "theme.docs.breadcrumbs.navAriaLabel": "頁面路徑",
```

**Review by Josh-Cena — APPROVED** [review]
Thanks

Waiting for @slorber so we can get fr translations as well!

File: packages/docusaurus-theme-translations/locales/fr/theme-common.json [hunk 3]
```diff
@@ -33,6 +33,8 @@
   "theme.common.headingLinkTitle": "Lien direct vers le titre",
   "theme.common.skipToMainContent": "Aller au contenu principal",
   "theme.docs.DocCard.categoryDescription": "{count} items",
+  "theme.docs.breadcrumbs.home": "Home page",
+  "theme.docs.breadcrumbs.navAriaLabel": "Breadcrumbs",
```
Comment (slorber) [inline packages/docusaurus-theme-translations/locales/fr/theme-common.json:37]:
```suggestion
  "theme.docs.breadcrumbs.home": "Page d'accueil",
  "theme.docs.breadcrumbs.navAriaLabel": "Fil d'Ariane",
```

**Pull request merged**